labs/lfdt/fabric-rust-integration.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: default
+title: Hyperledger Fabric Rust Integration
+[//]: # (parent: Labs)
+---
+# Lab Name
+Hyperledger Fabric Rust Integration
```

Comment:
I would suggest to name this lab something like "Fabric Chaincode Rust Integration", the current name made me thinking that this lab is about implementing fabric in Rust 🤓

Reply:
Yes you are right!